addons/account_peppol/tools/demo_utils.py

```diff
@@ -89,8 +89,11 @@ def _mock_button_verify_partner_endpoint(func, self, *args, **kwargs):
 def _mock_user_creation(func, self, *args, **kwargs):
     func(self, *args, **kwargs)
     self.account_peppol_proxy_state = 'receiver' if self.smp_registration else 'sender'
+    pkey = self.env['certificate.key'].create({
```

Comment:
I think you are creating a new certificate key everytime you send an invoice when being in demo mode